### DISCUSSION TITLE
Not showing up in HA Integrations

### DISCUSSION THREAD

**Discussion by grubis** [jason0x43/hacs-hubitat#109] [Q&A] [answered]
Hello Jason,

Thanks for your hard work on this integration. I just got a Hubitat as Smartthings discontinued support for older hardware... was time to make a switch.

I'm a long time user of HA and have a couple other custom components. I've tried both ways, using HACS to install the hubitat custom component and coping over the GIT repo and manually putting the hubitat directory in my custom components folder.

Where I get stuck is that I never see the custom component show up in Integrations (even in the "+" add integrations list). Have you run into this before? My other customer integrations are there (HACS being one of them).

Thanks for any pointers.

Best Regards,
Matt

**Comment by grubis** [comment, marked as answer]
So, I fixed it... I'll document here for others.
My mistake is that I was just restarting Home Assistant Core. When I restarted the entire Host, the custom component was available under adding integrations.

**Reply by jason0x43** [reply]
Thanks for posting a solution! A few other people have had that issue in the past, but I don't believe we ever determined the specific cause.

**Comment by cknic** [comment]
I'm still having this issue.  Trying to troubleshoot now.

**Comment by FirstRulez** [comment]
I am also having this issue; I have a new install of HACS on a Rasberry PI using the HAOS install, and I cannot see this addon either when logged on to the device itself or in the integrations list on the HACS website: https://www.home-assistant.io/integrations/#search/hubitat

Is the only option to install it manually now?

**Reply by jason0x43** [reply]
That list is for integrations built into Home Assistant, not those in HACS. The list of integrations included with HACS is here: https://hacs.xyz/default_repositories.

To add an integration with HACS, open the HACS entry in the sidebar, choose "Integrations", then click the "Explore & Add Repositories" button in the lower right. You should be able to find Hubitat in that list,

**Reply by jason0x43** [reply]
Regarding terminology (because it can get kind of confusing with HA), an "addon" is a feature installed using the Home Assistant Supervisor -- these are only available in a full Home Assistant install. Home Assistant also comes with a bunch of built in "integrations" -- these can be activated through Configuration -> Integrations, or through the `configuration.yaml` file.

HACS is itself an integration. You add it to HA, and it gives you easy access to a bunch of integrations that _aren't_ built into Home Assistant. To install those third party integrations (like this one), you can either use HACS or install them manually. It's all the same in the end, but HACS can let you know when updates are available.

**Reply by FirstRulez** [reply]
Thank you both. I was confused about HACS being different from HA itself. I have now installed HACS and managed to get the integration I wanted installed.

**Comment by rszimm** [comment]
I'm having a heck of a time here.  

I've got a docker install of HA.  I've put hubitat in my <config>/custom_components/hubitat directory.  I've installed the maker API.  I've rebooted my machine.  When I go to integrations and click the <+> button and search for Hubitat, nothing comes up.  I don't know what I've done wrong.  

Checking the logs I get the following: 
2022-10-23 08:30:24.325 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration hubitat which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant

So it seems HA is recognizing hubitat and seemingly loading it, but I cannot see it in the integrations ([see image here](https://i.imgur.com/FdtbsuR.png)

-----

I decided to scrap the manual install and download HACS and install via there and things worked.  One thing that occurred to me during the HACS install is they had you do a hard browser refresh, so I'm wondering if maybe that was the issue I was actually having.
